demos/m5/README.md

```diff
@@ -2,9 +2,19 @@
 
 This directory contains the resources needed for running the release demo for Milestone 5.
 
-## Environments
+# Bootstraping
 
-This demo can run in two different environments:
+After bootstrapping the cluster with App Studio, you can prepare the demo by running the [boostrap.sh](demos/m5/boostrap.sh) script.
+```
+$ git clone https://github.com/redhat-appstudio/release-service.git
+$ cd release-service/demos/m5
+$ ./boostrap.sh
```

Comment:
you really hate the first t in bootstrap huh

Reply:
I think it's time to get a new keyboard.